### ISSUE TITLE
AngularJs

### ISSUE THREAD

**Issue by Ruchika-Kuwar** [Ruchika-Kuwar/project-1#1]
<!DOCTYPE html>
<html lang="en" ng-app="myApp">
<head>
    <meta charset="UTF-8">
    <title>Simple AngularJs UI</title>
    <link rel="stylesheet" href="style.css">
</head>
<body ng-controller="FormController">
    <div class="box">
        <h2>User Form</h2>

        <label>Name:</label>
        <input type="text" ng-model="user.name" placeholder="Enter Name">

        <label>Email:</label>
        <input type="email" ng-model="user.gmail" placeholder="Enter email">

        <label>Gender:</label>
        <select ng-model="user.gender">
        <option value="">Select Gender</option>
        <option>Male</option>
        <option>Female</option>
        <option>Other</option>
        </select>

        <button ng-click="show = true">Submit</button>

        <div class="output" ng-if="show">
            <strong>Preview:</strong><br>
            Name: {{ user.name}}<br>
            Email: {{ user.gmail}}<br>
            Gender: {{ user.gender}}<br>
        </div>
    </div>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.8.2/angular.min.js"></script>
    <script src="style.js"></script>
</body>
</html>
